setup.cfg

```diff
@@ -18,18 +18,16 @@ classifiers =
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
     Programming Language :: Python
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
```

Comment:
Development Status :: 3 - Alpha

in the above classifiers, would be nice to also update the development status to alpha until we make a stable release

Reply:
yes, great suggestion, i'll update now